controllers/controller_common.go

```diff
@@ -37,6 +37,12 @@ var clusterRefLabel = "kafkaCluster"
 // use as var so it can be overwritten from unit tests
 var newKafkaFromCluster = kafkaclient.NewFromCluster
 
+func requeueAfter(sec int) (ctrl.Result, error) {
```

Comment:
The return error is needed as this function is used to end the reconcile flow and the controller runtime expects two values to be return where the first is a `ctrl.Result` and the second is an error which happens to be `nil` in this case.

So you can do this:
```
return requeueAfter(5)
```
instead of this:
```
return requeueAfter(5), nil
```
or as it was used previously
```
return ctrl.Result{
    RequeueAfter: time.Duration(5)*time.Second
}, nil
```